Dockerfile

```diff
@@ -123,3 +123,94 @@ RUN vast exec version
+      rm -rf build-compaction
+
+FROM plugins-source AS inventory-plugin
+
+RUN cmake -S contrib/vast-plugins/inventory -B build-inventory -G Ninja \
+      -D CMAKE_INSTALL_PREFIX:STRING="$PREFIX" && \
+      cmake --build build-inventory --parallel && \
+      DESTDIR=/plugin/inventory cmake --install build-inventory --strip --component Runtime && \
+      rm -rf build-inventory
+
+FROM plugins-source AS matcher-plugin
+
+RUN cmake -S contrib/vast-plugins/matcher -B build-matcher -G Ninja \
+      -D CMAKE_INSTALL_PREFIX:STRING="$PREFIX" && \
+      cmake --build build-matcher --parallel && \
+      DESTDIR=/plugin/matcher cmake --install build-matcher --strip --component Runtime && \
+      rm -rf build-matcher
+
+FROM plugins-source AS netflow-plugin
+
+RUN cmake -S contrib/vast-plugins/netflow -B build-netflow -G Ninja \
+      -D CMAKE_INSTALL_PREFIX:STRING="$PREFIX" && \
+      cmake --build build-netflow --parallel && \
+      DESTDIR=/plugin/netflow cmake --install build-netflow --strip --component Runtime && \
+      rm -rf build-netflow
+
+FROM plugins-source AS pipeline-manager-plugin
+
+RUN cmake -S contrib/vast-plugins/pipeline_manager -B build-pipeline_manager -G Ninja \
+      -D CMAKE_INSTALL_PREFIX:STRING="$PREFIX" && \
+      cmake --build build-pipeline_manager --parallel && \
+      DESTDIR=/plugin/pipeline_manager cmake --install build-pipeline_manager --strip --component Runtime && \
+      rm -rf build-pipeline_manager
+
+FROM plugins-source AS platform-plugin
+
+RUN cmake -S contrib/vast-plugins/platform -B build-platform -G Ninja \
+      -D CMAKE_INSTALL_PREFIX:STRING="$PREFIX" && \
+      cmake --build build-platform --parallel && \
+      DESTDIR=/plugin/platform cmake --install build-platform --strip --component Runtime && \
+      rm -rf build-platform
+
+# -- vast-ce -------------------------------------------------------------------
+
+FROM production AS vast-ce
+
+COPY --from=matcher-plugin --chown=vast:vast /plugin/matcher /
+COPY --from=netflow-plugin --chown=vast:vast /plugin/netflow /
+COPY --from=pipeline-manager-plugin --chown=vast:vast /plugin/pipeline_manager /
+COPY --from=platform-plugin --chown=vast:vast /plugin/platform /
+
+# -- vast-demo --------------------------------------------------------------
+
+FROM vast-ce AS vast-demo
+
+USER root:root
+COPY demo-node /demo-node
+RUN apt-get update && \
+    apt install -y \
+        curl \
+        bc \
+        zstd && \
+        rm -rf /var/lib/apt/lists/*
+
+ENTRYPOINT ["/demo-node/entrypoint.bash"]
+
+# -- vast-ee -------------------------------------------------------------------
+
+FROM vast-ce AS vast-ee
+
+COPY --from=compaction-plugin --chown=vast:vast /plugin/compaction /
+
+# -- vast ----------------------------------------------------------------------
+
+# As a last stage we re-introduce VAST without proprietary plugins so that it's
+# the default when not specifying a build target.
+FROM production AS vast
```

Comment:
Can you switch from the `production` target to `vast` in `docker-config-base.json`, `docker.yaml`, and `docker-manual.yaml` in the workflows directory?

Reply:
Done in 989d4a5b40